File: cses/introductory/cse_increasing_array.py
n = int(input())
count = 0
arr = list(map(int, input().split()))
for k in range(len(arr)-1):
    if arr[k] > arr[k+1]:
        dif = arr[k] - arr[k+1]
        arr[k+1] += dif
        count += dif
print(count)    


        

File: cses/introductory/cse_increasing_array.cpp
#include <iostream>
#include <stdlib.h>
#include <vector>

using namespace std;

int main () {

    int n, j, dif;
    int count = 0;
    vector<int> v;

    cin >> n; 
    for (int i = 0; i < n; i++) {
        cin >> j;
        v.push_back(j);
    }

    for (int i = 0; i < n; i++) {
        if (v[i] > v[i+1]) {
            dif = v[i] - v[i+1];
            v[i+1] = dif;
            count += dif;
        }
    }

    cout << count << endl;


    return 0;

}